src/js/view/view.js

```diff
@@ -716,6 +724,17 @@ define([
                 default:
                     break;
             }
```

Comment:
How about this?

```
            if (state === states.COMPLETE) {
                _api.setFullscreen(false);
            }
            
            switch (state) {
                case states.IDLE:
                case states.ERROR:
                case states.COMPLETE:
                    _captionsRenderer.hide();
                    break;
                case states.PAUSED:
                    _captionsRenderer.show();
                    if (_controls && !_controls.showing) {
                        _captionsRenderer.renderCues(true);
                    }
                 break;
                default:
                    _captionsRenderer.show();
                    break;
            }
```

I like breaking out the fullscreen check because then the switch case is just related to the captions logics. And this way, we may be dupe call to `_captionsRenderer.show();`, which is clearer and avoids doing a state check in `default` which doesn't feel right.